client/lib/plans/features-list.tsx

```diff
@@ -1667,6 +1671,22 @@ export const FEATURES_LIST: FeatureList = {
 		getSlug: () => FEATURE_LINK_IN_BIO_THEMES_CUSTOMIZATION,
 		getTitle: () => i18n.translate( 'Advanced link in bio themes and customization' ),
 	},
+	[ FEATURE_NEWSLETTER_IMPORT_SUBSCRIBERS_FREE ]: {
+		getSlug: () => FEATURE_NEWSLETTER_IMPORT_SUBSCRIBERS_FREE,
+		getTitle: () => i18n.translate( 'Import up to 100 subscribers' ),
+	},
+	[ FEATURE_PAYMENT_TRANSACTION_FEES_8 ]: {
+		getSlug: () => FEATURE_PAYMENT_TRANSACTION_FEES_8,
+		getTitle: () => i18n.translate( '8% transaction fee for payments + Stripe fees' ),
+	},
+	[ FEATURE_PAYMENT_TRANSACTION_FEES_4 ]: {
+		getSlug: () => FEATURE_PAYMENT_TRANSACTION_FEES_4,
+		getTitle: () => i18n.translate( '4% transaction fee for payments + Stripe fees' ),
+	},
+	[ FEATURE_PAYMENT_TRANSACTION_FEES_2 ]: {
+		getSlug: () => FEATURE_PAYMENT_TRANSACTION_FEES_2,
+		getTitle: () => i18n.translate( '2% transaction fee for payments + Stripe fees' ),
+	},
```

Comment:
Should these features (or some variation of them) also appear in the comparison table at the bottom?

Reply:
cc @lucasmendes-design What features should the plans comparison grid listing for the newsletter flow?